src/state/CalculationLogics.ts

```diff
@@ -0,0 +1,219 @@
+import i18n from "i18next";
+import { Cycle } from "../data/ClassCycle";
+
+const millisecondsInDay = 24 * 60 * 60 * 1000;
+
+export function getOvulationStatus(cycleLength: number, dayOfCycle: number) {
+  if (!cycleLength || !dayOfCycle) {
+    return "";
+  }
+
+  const lutealPhaseLength = 14;
+  const ovulationDay = cycleLength - lutealPhaseLength;
+  const diffDay = ovulationDay - dayOfCycle;
+  if (diffDay === 0) {
+    return i18n.t("today");
+  } else if (diffDay < 0 && diffDay >= -2) {
+    return i18n.t("possible");
+  } else if (diffDay < 0) {
+    return i18n.t("finished");
+  } else if (diffDay === 1) {
+    return i18n.t("tomorrow");
+  } else if (diffDay < 5) {
+    return `${i18n.t("in")} ${diffDay} ${i18n.t("Days less 5")}`;
```

Comment:
With days_interval you can delete this expression

Reply:
Fixed